### DISCUSSION TITLE
Community input wanted: Should onion skins be enabled for all layers or only the selected layer?

### DISCUSSION THREAD

**Discussion by keiwando** [keiwando/toonsquid#481] [Polls]
Do you like the fact that ToonSquid currently shows onion skins for all of your layers by default when you tap the onion skin button and you have to manually disable onion skinning on individual layers afterwards or would you prefer it if it was the other way round and it only showed onion skins of the selected layer by default?

**Comment by EdaliaDay** [comment]
I would love it if it were implemented to make it just the selected layer. It's annoying to currently disable on lots of layers separately
